python/idsse_common/idsse/common/rabbitmq_utils.py

```diff
@@ -446,8 +447,32 @@ def __init__(
         self._exch = exch_params
         self._queue = None
 
-        self.connection = BlockingConnection(conn_params.connection_parameters)
-        self.channel = self.connection.channel()
+        # establish RabbitMQ connection/channel and initialize exchange (or reuse provided channel)
```

Comment:
I know I suggested having a setup type function, but with how minor your changes had to be, does it make sense?